features/tasks.feature

```diff
@@ -1360,6 +1367,7 @@ Feature: Tasks
             "hasIncidents": false,
             "incidents": [],
             "packages": [],
+            "barcode": "@array@",
```

Comment:
barcodes should also be returned on packages right? or we will add it later

Reply:
It already the case, the packages barcodes are returned. But you need to enable the group `barcode` when serializing the `Task` entity. I did it this way to avoid rendering all the barcodes when you call `GET /api/tasks` since this endpoint is critical and already a bit slow